frontend/view-edit-lead/lead-home/view-edit-lead-services-info.component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { SingleLead } from "../view-lead.component";
+import LeadSection from "./lead-section.component";
+
+export default function ViewEditLeadServicesInfo(
+  props: ViewEditLeadServicesInfoProps
+) {
+  const { lead } = props;
+
+  return (
+    <LeadSection title="Services of Interest">
+      <table className="lead-service-table">
+        <thead>
+          <tr>
+            <th>Name of Service</th>
+            <th>Program</th>
+            <th>Status</th>
+          </tr>
+        </thead>
+        <tbody>
+          {lead.leadServices.map(service => {
+            return (
+              <tr>
```

Comment:
This will cause a warning in the console because no `key` prop is provided:

```suggestion
              <tr key={service.id}>
```